src/Admin/FieldDescription.php

```diff
@@ -88,7 +80,7 @@ public function setParentAssociationMappings(array $parentAssociationMappings):
         $this->parentAssociationMappings = $parentAssociationMappings;
     }
 
-    public function isIdentifier()
+    public function isIdentifier(): bool
```

Comment:
This return type hints should be remove from AdminBundle to allow use master admin with Sonata3 bundles. Some others change too. Mybe we should w8 with this PR and remove some type hints from Admin first?

Reply:
https://github.com/sonata-project/dev-kit/issues/780